### PR TITLE
Converts map object to list (Python 3 compatibility)

### PR DESCRIPTION
**Pull request by amarvin**

### CODE REVIEW

File: django_hana/operations.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def modify_insert_params(self, placeholder, params):
         insert_param_groups = []
         for p in params:
             if isinstance(p, list):
-                insert_param_groups.append(map(self.sanitize_bool, p))
+                insert_param_groups.append(list(map(self.sanitize_bool, p)))
```
Comment (mathebox) [inline django_hana/operations.py]: I would prefer a list comprehension:
```
[self.sanitize_bool(value) for value in p]
```

**Review by mathebox — CHANGES_REQUESTED** [review]
It would be great if you could also provide a test which checks this line. This can be a general case or the query that's similar to your. Have a look at [`tests/test_queries.py`](https://github.com/mathebox/django_hana_pyhdb/blob/fe52f8f05781d08293d18200c418bc1a07c3eb4b/tests/test_queries.py)

**Comment by amarvin** [comment]
I converted it to list comprehension, but haven't added a test yet. I'm quite new to this.

**Comment by amarvin** [comment]
I took a couple cracks at writing a test, but failed, sorry!

**Comment by amarvin** [comment]
What was I doing wrong in my test? I get a strange database connection error in Travis-CI on the .save().

**Comment by mathebox** [comment]
On travis the test tries to connect to a SAP HANA instance. This is not possible.
Creating a mock for `BaseDatabaseSchemaEditor.execute` is necessary when testing methods that alter the schema of the table (e.g. migration). An `INSERT` query will need a different mock.
I will look into this.

**Comment by amarvin** [comment]
I tried modifying the test with a mock of the .save command, as you mentioned, using this as an example: https://stackoverflow.com/questions/10545049/mocking-django-model-and-save. I was unsuccessful, sorry.

**Comment by mathebox** [comment]
This should fixed by 47f029e75140ffba0504d5a85c6f3fe55d36b52b. I have also improved the mock behavior of the tests. This should make the tests better understandable. Have a look.
If you have further questions or bugs, please create an issue.

**Comment by amarvin** [comment]
Terrific, thanks!

**Pull request closed without merging**